addons/account/views/account_move_views.xml

```diff
@@ -1309,10 +1309,10 @@
                     <separator/>
                     <filter string="To Check" name="to_check" domain="[('to_check', '=', True)]"/>
                     <separator/>
-                    <!-- in_payment & not_paid & partial -->
-                    <filter name="open" string="Open" domain="[('state', '=', 'posted'), ('payment_state', 'in', ('in_payment', 'not_paid', 'partial'))]"/>
-                    <!-- paid & reversed -->
-                    <filter name="closed" string="Closed" domain="[('state', '=', 'posted'), ('payment_state', 'in', ('paid', 'reversed'))]"/>
+                    <!-- not_paid & partial -->
+                    <filter name="open" string="Unpaid" domain="[('state', '=', 'posted'), ('payment_state', 'in', ('not_paid', 'partial'))]"/>
+                    <!-- in_payment & paid -->
+                    <filter name="closed" string="Paid" domain="[('state', '=', 'posted'), ('payment_state', 'in', ('in_payment', 'paid'))]"/>
```

Comment:
reversed should be considered closed as well

Reply:
@tsb-odoo @avw-odoo you guys don't want reversed invoices to appear under the 'paid' filter? I find it weird, but hey... you know better :shrug:

Reply:
@qdp-odoo @avw-odoo indeed, there's no money transfer. It's not strictly a payment, functionally speaking. IMHO

Reply:
@tsb-odoo @avw-odoo @qdp-odoo IMHO "paid" is not an indication of payments but rather "there is nothing due" so it should include records settled by credit notes/refunds/ reversed

Reply:
> "paid" is not an indication of payments

![WAT](https://static.wikia.nocookie.net/memepediadankmemes/images/c/cc/Wat8.jpg/revision/latest/top-crop/width/360/height/450?cb=20180913191230)

Reply:
`account.payment`